backend/store/task.go

```diff
@@ -190,7 +190,7 @@ func (s *Store) ListTasks(ctx context.Context, find *api.TaskFind) ([]*TaskMessa
 		where = append(where, fmt.Sprintf("task.status in (%s)", strings.Join(list, ",")))
 	}
 	if v := find.LatestTaskRunStatusList; v != nil {
-		where = append(where, fmt.Sprintf("latest_task_run.status = ANY($%d)", len(args)+1))
+		where = append(where, fmt.Sprintf("COALESCE(latest_task_run.status, 'NOT_STARTED') = ANY($%d)", len(args)+1))
```

Comment:
Use TaskRunNotStarted instead of hardcoded "NOT_STARTED"?

Reply:
Done.